File: template/part2.py
def part2(x):
	with open(x, "r") as f:
		data = f.read().splitlines()
	val = 0
	return val

test_val = 0
if part2("test") == test_val:
	print('Solution is {}'.format(part2("input")))
else:
	print("Test failed")

File: template/rust/src/part2.rs
fn solution(_data: String) -> u32 {
    0
}

pub fn print_solution(test_val: u32) {
    let input = std::fs::read_to_string("./input").unwrap();
    let test = std::fs::read_to_string("./test").unwrap();

    if solution(test) != test_val {
        println!("Test failed");
    } else {
        println!("{}", solution(input));
    }
}
